code/modules/client/preference_setup/vore/07_traits.dm

```diff
@@ -313,6 +335,63 @@
 			pref.custom_exclaim = exclaim_choice
 		return TOPIC_REFRESH
 
+	else if(href_list["custom_heat"])
+		var/numMessages = pref.custom_heat.len
+		if(numMessages > 0)
+			var/alert = tgui_alert(user, "Edit current messages or add a new one?","Selection",list("Edit","Add", "Cancel"))
+			switch(alert)
+				if("Edit")
+					var/list/indices = list()
+					var/i
+					for(i=1, i<= numMessages, i++)
+						indices.Add(i)
```

Comment:
I'm uncertain about need for this to be an indexed list at all. Something more akin to how bellies handle it where its just a singular text list of descriptions with double linebreaks as separator would work better here. Main reason I used indexing for flavor texts of nutrition and fatness was due to the fact that you both HAD to have all 10 and because the order was very important, unlike here where neither amount you have nor the order matters.

Reply:
Requested changes done.

I did not make it into a whole proc of its own as that seems far too involved for something so minor.